src/containers/Checkout/Checkout.js

```diff
@@ -49,20 +49,38 @@ export default class Checkout extends Component {
       this.setState({preparedOrderItems: this.preparedOrderItems(nextProps.orderItems, nextProps.user)});
     }
   }
-
+  /**
+   * @description This function returns deliveries amount.[Ф-ция возращает количество доставок.]
+   * @param items Array of objects "Lunch". [Массив объектов "Lunch".]
+   * @returns {*}
+     */
   purchasedTariffsCount(items) {
     return items.reduce((result, item) => result + (item.resource_type === 'DeliveryTariff' ? item.resource.amount : 0), 0);
   }
 
+  /**
+   * @description This function prepares array of lunches for displaying in order list. [Ф-ция подготавлевает массив "Lunch" для вывода в списке заказа.]
+   * @param items Array of objects "Lunch". [Массив объектов "Lunch".]
+   * @param user User. [Залогиненый или не залогиненый пользователь.]
+   * @returns {{purchasing: null[], grouped}}
+     */
   preparedOrderItems(items, user) {
     const { tariffs } = this.props;
     const groupedItems = this.groupedItems(items);
     const individualTariffItem = orderItemStructure('DeliveryTariff', find(tariffs, {individual: true}));
     const zeroTariffItem = orderItemStructure('DeliveryTariff', { price: 0, zero: true });
     const individualTariffItems = [];
-
+    /**
+     * @description Current variable receives deliveries amount of current user. [Количество доставок у пользователя.]
```

Comment:
This variable represents deliveries amount for current order including user's deliveries available and purchasing deliveries in order